02-intermediate-ruby/object-inheritance.md

```diff
@@ -53,6 +53,8 @@ fun_place.mailing_address #=> "111 Boardwalk"
                           #   "New York, NY 12070"
 ```
 
+> **Exercise**: Create a `Shape` class with a `color` attribute and a `describe` method. Create a `Circle` class as a child.
```

Comment:
This exercise is immediately underneath "Inheriting Methods," and it just has them creating two classes. Is there a stronger way we can connect the two concepts? Is there a way they can verify that they're on the right track, with the right syntax?

Reply:
Yes, great point. I'll add instructions for that. Thanks!